__tests__/app.test.js

```diff
@@ -378,4 +352,27 @@ describe("GET /api/articles?topic", () => {
             })
         })
     })
+})
+
+describe("GET /api/articles/:article_id (comment_count)", () => {
+    test("status 200: responds with a specified article with added property of comment_count", () => {
```

Comment:
This test can and should live in the describe block for get /api/articles/:article_id